docs/t-sql/functions/text-and-image-functions-textvalid-transact-sql.md

```diff
@@ -62,8 +62,8 @@ TEXTVALID ( 'table.column' ,text_ ptr )
   
 |Function or statement|Description|  
 |---------------------------|-----------------|  
-|PATINDEX**(**'_%pattern%_'**,** _expression_**)**|Returns the character position of a specified character string in **text** and **ntext** columns.|  
-|DATALENGTH**(**_expression_**)**|Returns the length of data in **text**, **ntext**, and **image** columns.|  
+|PATINDEX<b>('</b>_%pattern%_'**,** _expression_**)**|Returns the character position of a specified character string in **text** and **ntext** columns.|  
+|DATALENGTH<b>(</b>_expression_**)**|Returns the length of data in **text**, **ntext**, and **image** columns.|  

```

Comment:
Thank you for catching this! Per our style guide, expressions and parameters should be italicized and not bolded, so I believe the bolding itself was incorrect. This will remove the typo and keep the italics, plus add a little additional spacing for readability:

```suggestion
|PATINDEX ( *'_%pattern%_'*, *_expression_* )|Returns the character position of a specified character string in **text** and **ntext** columns.|  
|DATALENGTH ( *_expression_* )|Returns the length of data in **text**, **ntext**, and **image** columns.| 
```

Reply:
@LitKnd - Thank you for your suggestion.

Reply:
GitHub should give you a button to accept the suggestion, and then give you a commit button, [like in this doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes).

It looks like this is getting marked as resolved here but the commit isn't going through?

Reply:
And thank you :)

Reply:
@LitKnd - Thanks! I pressed the commit button.